PI_Log.txt.1

```diff
@@ -0,0 +1,2 @@
+2017-04-17 23:49:00 Pi SerialOutputListener created
```

Comment:
This file shouldn't have been checked in. We should add it to the .gitignore file. 

It won't let me comment on it directly, but there is also an image which was checked in that shouldn't be. We need to add the entire `PI_PICS` directory to the .gitignore file as well.